examples/ChaChaPoly/chacha_poly.ec

```diff
@@ -80,14 +80,17 @@ theory C.
 
   axiom gt0_max_counter : 0 < max_counter.
 
+  type counter.
+
   clone include Subtype with 
```

Comment:
Should this (and line 124) be `subtype` commands?

Reply:
Yes, I forgot to do the examples.